beacon-chain/blockchain/service.go

```diff
@@ -299,7 +325,63 @@ func (c *ChainService) processBlock(block *types.Block) error {
 	return nil
 }
 
-func (c *ChainService) executeStateTransition(
+func (c *ChainService) processBlock(block *types.Block) error {
```

Comment:
Please add godoc comment with the spec definition for easier understanding. 
Please use a snippet from the beacon_chain.md spec.